front/src/modules/ui/board/states/boardColumnTotalsFamilySelector.ts

```diff
@@ -1,25 +1,22 @@
 import { selectorFamily } from 'recoil';
 
 import { companyProgressesFamilyState } from '@/companies/states/companyProgressesFamilyState';
-import { BoardPipelineStageColumn } from '@/ui/board/components/Board';
 
-import { boardState } from './boardState';
+import { boardCardIdsByColumnIdFamilyState } from './boardCardIdsByColumnIdFamilyState';
 
+// TODO: this state should be computed during the synchronization hook and put in a generic
```

Comment:
So you recommend we create and maintain the sums as a separated piece of state ?
I thought it'd be painful to have to keep it up to date at each board operation, hence the derived state approach using selector.

Reply:
The problem here is that we have to use progress to compute the derived state and we want the column sum to be an agnostic number.